VAMobile/src/store/api/api.ts

```diff
@@ -18,6 +22,10 @@ let _store: ReduxToolkitStore | undefined
 
 const DEMO_MODE_DELAY = 300
 const METHODS_THAT_ALLOW_PARAMS = ['GET']
+// @ts-expect-error
+const DEVICE_MODEL = Platform.OS === 'ios' ? 'iPhone' : Platform.constants.Model
+// @ts-expect-error
+const OS_VERSION = Platform.OS === 'ios' ? `iOS ${Platform.Version}` : `Android ${Platform.constants.Release}`
```

Comment:
Had to tell typescript to ignore/expect an error from this because for some reason attributes of the `Platform.constants` object aren't being recognized by TypeScript even though they do exist ([React Native docs](https://reactnative.dev/docs/0.73/platform#constants))